metadata.rb

```diff
@@ -5,8 +5,10 @@
 description      'Installs/Configures magento-ng'
 long_description 'Installs/Configures magento-ng'
 version          '0.4.1'
+source_url       'https://github.com/inviqa/chef-magento-ng'
+issues_url       'https://github.com/inviqa/chef-magento-ng/issues'
 
 depends 'config-driven-helper', '~> 1.5'
-depends 'cron', '~> 1.4'
+depends 'cron', '~> 1.7'
 depends 'logrotate', '~> 1.9'
-depends 'php-fpm'
+depends 'php-fpm', '~> 0.7'
```

Comment:
I might prefer php-fpm version be less restricted given the very loose coupling between the two (just uses the pool's user attribute)